Source/UI/Common/Styles/ButtonStyle.swift

```diff
@@ -37,34 +37,34 @@ public struct ButtonStyle {
     // MARK: - Regular button
     
     /// If set, then color is applied to `UIButton.tintColor`
-    public let tintColor: UIColor?
+    public var tintColor: UIColor?
```

Comment:
Idea behind this is so that we can easily copy button styles

```
let style1 = ButtonStyle(...)
var style2 = style1
style2.borderWidth = 5
// now we have 2 same styles with just different border width
```